astropy/modeling/tests/test_models.py

```diff
@@ -335,6 +335,7 @@ def setup_class(self):
         self.y1 = np.arange(1, 10, .1)
         self.y2, self.x2 = np.mgrid[:10, :8]
 
+    @pytest.mark.filterwarnings(r'ignore:.*:RuntimeWarning')
```

Comment:
For some reason I can't find a test log that shows the warnings. Can you point me to one. I'm curious what raises this `RuntimeWarning`.

Reply:
You can see it in the log of #7928. For example, starting at https://travis-ci.org/astropy/astropy/jobs/608405975#L2929